app/controllers/api_controller/tags.rb

```diff
@@ -7,7 +7,8 @@ module Tags
     #
 
     def tags_query_resource(object)
-      object ? object.tags.where(Tag.arel_table[:name].matches "#{TAG_NAMESPACE}%") : {}
+      # object ? object.tags.where(Tag.arel_table[:name].matches "#{TAG_NAMESPACE}%") : {}
```

Comment:
I cannot do that, on https://github.com/ManageIQ/manageiq/blob/master/app/controllers/api_controller/renderer.rb#L193 `/managed` is the default namespace.

Chargeback use `/assigned_to` namespace :/